docs/userguide.rst

```diff
@@ -221,3 +221,31 @@ were deployed during ``init``, you can run:
 .. code-block:: console
 
    $ reana-cluster down
+
+Delete interactive sessions
+---------------------------
+
+Interactive sessions that were not closed after work have been finished leave
+active Kubernetes objects (pod, service and ingress) running in the Reana cluster.
+These objects might need to be deleted from time to time. Please note that
+interactive session pod and service are linked to the ingress object. If ingress
+object gets deleted other twos will be deleted automatically.
```

Comment:
What do you think about this?
"Please note that all interactive session Kubernetes objects are linked to the Ingress, so if it is deleted all dependent objects will also be deleted."

We might need more Kubernetes objects for interactive sessions at some point (like `configmaps`).

Reply:
I think this would make documentation less understandable, because user can assume that objects of all interactive sessions are linked to the one ingress object, kind of N:1 relationship, while actually it is 1:1.